packages/wrangler/src/d1/migrations/apply.ts

```diff
@@ -49,9 +49,17 @@ export const d1MigrationsApplyCommand = createCommand({
 				"Specify directory to use for local persistence (you must use --local with this flag)",
 			requiresArg: true,
 		},
+		"auto-apply": {
+			type: "boolean",
+			description: "Automatically apply all pending migrations without prompt",
+			default: false,
+		},
```

Comment:
I don't think we want to enable this only at the D1 level. There were few discussions internally for a "non-interactive" mode for wrangler, and therefore it's better to have a more consistent plan across products instead of doing something just for D1 migrations.